rc/lsp.kak

```diff
@@ -1973,6 +2011,21 @@ map global lsp-selection-range b '<esc>: lsp-selection-range-select bottom<ret>'
 map global lsp-selection-range t '<esc>: lsp-selection-range-select top<ret>'    -docstring 'select outermost node'
 
 define-command -hidden lsp-mappings-enable -params 1 -docstring "Add LSP mappings to goto and object mode" %{
+    map %arg{1} insert <tab> '<a-;>: lsp-snippets-select-next-placeholders-or <lt>tab><ret>' -docstring 'Select next snippet placeholder'
+    map %arg{1} normal <tab> ': lsp-snippets-select-next-placeholders-or <lt>tab><ret>' -docstring 'Select next snippet placeholder'
+
+    map %arg{1} insert <c-i> '<a-;>: lsp-snippets-select-next-placeholders-or <lt>c-i><ret>' -docstring 'Select next snippet placeholder'
+    map %arg{1} normal <c-i> ': lsp-snippets-select-next-placeholders-or <lt>c-i><ret>' -docstring 'Select next snippet placeholder'
+
+    hook -group lsp window InsertCompletionShow .* "
+      unmap %arg{1} insert <c-n> '<a-;>: lsp-snippets-select-next-placeholders<ret>'
+    "
+    hook -group lsp window InsertCompletionHide .* "
+      map %arg{1} insert <c-n> '<a-;>: lsp-snippets-select-next-placeholders<ret>' -docstring 'Select next snippet placeholder'
```

Comment:
I don't like this specific option because it doesn't fix the underlying problem that keys like `goto r` are still mapped every time we run `lsp-enable`.

Reply:
sounds good!